Sources/Brave/Frontend/Browser/BrowserViewController/BrowserViewController+TabManagerDelegate.swift

```diff
@@ -232,16 +232,20 @@ extension BrowserViewController: TabManagerDelegate {
     }
 
     let openNewTab = UIAction(
-      title: PrivateBrowsingManager.shared.isPrivateBrowsing ? Strings.Hotkey.newPrivateTabTitle : Strings.Hotkey.newTabTitle,
-      image: PrivateBrowsingManager.shared.isPrivateBrowsing ? UIImage(systemName: "plus.square.fill.on.square.fill") : UIImage(systemName: "plus.square.on.square"),
+      title: privateBrowsingManager.isPrivateBrowsing ? Strings.Hotkey.newPrivateTabTitle : Strings.Hotkey.newTabTitle,
+      image: privateBrowsingManager.isPrivateBrowsing ? UIImage(systemName: "plus.square.fill.on.square.fill") : UIImage(systemName: "plus.square.on.square"),
       handler: UIAction.deferredActionHandler { [unowned self] _ in
-        self.openBlankNewTab(attemptLocationFieldFocus: false, isPrivate: PrivateBrowsingManager.shared.isPrivateBrowsing)
+        self.openBlankNewTab(attemptLocationFieldFocus: false, isPrivate: privateBrowsingManager.isPrivateBrowsing)
       })
 
     if (UIDevice.current.userInterfaceIdiom == .pad && tabsBar.view.isHidden) || (UIDevice.current.userInterfaceIdiom == .phone && toolbar == nil) {
       newTabMenuChildren.append(openNewTab)
     }
     addTabMenuChildren.append(openNewTab)
+    
+    addTabMenuChildren.append(UIAction(title: "New Window", image: UIImage(systemName: "window.horizontal.closed"), handler: UIAction.deferredActionHandler { [unowned self] _ in
```

Comment:
Localize

Reply:
This is not the official way of opening multiple windows so I didn't localize it.

I basically just need a small design icon or placement of the icon somewhere then I'll wire it.

Atm, long pressing on the plus sign doesn't seem so good, but I didn't know where else to put it :D

Reply:
We should probably mimic Safari where we can:

- Link context menus: "Open in New Window"
- Long press on tab tray: "New Window" (This is where they have New Tab/New Private Tab though, so for us it would be long-press on + button)
- Drag and Drop, though not sure if we can mimic this 100%